deploy/kubernetes/driver/kubernetes/manifests/setup-vpc-file-sa.yaml

```diff
@@ -7,6 +7,7 @@
    - name: bluemix-default-secret
```

Comment:
can we create a new GHE to cleanup deployment and more restricted as per requirement for file and ready for Nov release